lib/jobba/status.rb

```diff
@@ -6,6 +6,14 @@ class Status
 
     include Jobba::Common
 
+    def self.all
+      Query.new
+    end
+
+    def self.where(*args)
+      all.where(*args)
+    end
```

Comment:
Can `all` and `where` stay just in `Jobba`?  I'm ok with `create` and `find` being in `Status` and having delegators in `Jobba` for convenience, because those methods return `Status`.  Adding `all` and `where` to `Status` adds a dependence on `Query` that I'd rather not have.  Let me know if there's a specific reason to keep it this way.

If you'd rather keep the Jobba implementation be all delegators, maybe something like:

```ruby
module Jobba
  class << self
    extend Forwardable

    def_delegator Jobba::Query, :new, :all
    def_delegators Jobba::Query, :where
    def_delegators Jobba::Status, :create, :create!, :find, :find!
  end
end

class Jobba::Query
  def self.where(...)
    new.where(...)
  end
  ...
end
```